packages/contracts/scripts/deploy-scripts/mainnet-genesis.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+export L2_BLOCK_GAS_LIMIT=15000000
+export L2_CHAIN_ID=10
+export BLOCK_SIGNER_ADDRESS=0x00000398232E2064F896018496b4b44b3D62751F
+export L1_STANDARD_BRIDGE_ADDRESS=0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1
+export L1_FEE_WALLET_ADDRESS=0x391716d440c151c42cdf1c95c1d83a5427bca52c
+export L1_CROSS_DOMAIN_MESSENGER_ADDRESS=0x25ace71c97B33Cc4729CF772ae268934F7ab5fA1
+export WHITELIST_OWNER=0x648E3e8101BFaB7bf5997Bd007Fb473786019159
+export GAS_PRICE_ORACLE_OWNER=0x648E3e8101BFaB7bf5997Bd007Fb473786019159
```

Comment:
This is the only value that I'm not confident in

Reply:
verified that this matches the k8s settings, but not sure if it should be different

Reply:
We shouldn't change the gas oracle owner, so its good